src/src/com/microsoft/aad/adal/AuthenticationSettings.java

```diff
@@ -44,7 +44,11 @@
      */
     private String mSharedPrefPackageName;
 
-    private boolean mSkipBroker = false;
+    /**
+     * @deprecated As of release 1.1.14, replaced by {@link #mUseBroker}
+     */
+    @Deprecated private boolean mSkipBroker = false;
+    private boolean mUseBroker = true;
```

Comment:
yup, today we set skipBroker to be false to use broker by default. I'm not quite sure if this is also something we want to change.